### DISCUSSION TITLE
コメントのリンクが日時クリックなのがわかりづらい

### DISCUSSION THREAD

**Discussion by uhooi** [increments/qiita-discussions#28] [要望・アイデア・不具合]
記事のコメントのリンクが、コメントの日時をクリックしないと取得できないのが非常にわかりづらいです。
「コメントのリンクを取得したい → 日時の横にある「…」をクリック → 「通報」しかない → 困った → ユーザー名をクリックしてみる → ユーザーのページに飛ぶ → 詰んだ」のループを何回繰り返したことか…ｗ
<img width="841" alt="スクリーンショット_2021-10-06_10_44_09" src="https://user-images.githubusercontent.com/21194714/136127674-76f1e699-669c-4e08-a6bb-600f2cb02f41.png">

「…」メニューに「コメントのリンクを取得」を追加してほしいです。
他にわかりやすい方法があればそれでも問題ありません。

**Comment by kskwtnk** [comment]
@uhooi 

ご意見ありがとうございます！

運営がこんなことを言うのもどうかと言う話なんですが、私たち自身もかなり迷ってしまうリンクです
仰るとおり「…」の中にリンク取得がある方が確実に分かりやすいというか、世間のUIと足並みが揃っているというかなので、対応します！

**Comment by ohakutsu** [comment]
このDiscussionから作成されたIssueの [コメントのリンクが日時クリックなのがわかりづらい Issue #40](https://github.com/qiita-discussions/qiita-discussions/issues/40) の対応が完了しました！

**Reply by uhooi** [reply]
ありがとうございます…！
ディスカッションを閉じたいのですが、メンテナーしか閉じられないでしょうか…？

**Reply by kskwtnk** [reply]
@uhooi Discussionなのですが、現在はロールに関わらず閉じられないようになっているようです……
一旦、ラベルなどの運用でカバーする方法を考えますので少しお待ちください！
